app/controllers/taxons_controller.rb

```diff
@@ -0,0 +1,37 @@
+class TaxonsController < ApplicationController
+  before_action :redirect_to_www, unless: :new_navigaton_enabled?
+
+  def show
+    render :show, locals: {
```

Comment:
@tijmenb this commit reduces the locals being passed to the view: https://github.com/alphagov/collections/pull/217/commits/f6676fe42050f2f06e4c73f49560481da920ba3f